README.md

```diff
@@ -42,6 +47,7 @@ As a React.js developer, you just write a few lines of code get a live notebook
 
 Jupyter React can be used in any React.js application. Install the latest `@datalayer/jupyter-react` npm package and get inspired by the [examples](https://github.com/datalayer/jupyter-react/tree/main/examples) in this repository.
 
+
```

Comment:
My big question reading this is: "How do I run the notebook server alongside my react app".  presumably you have some node/express server servingup your react app, that connects toa  running notebook server, how?

Reply:
The readme should highlight a server is needed. jupyter server runs on python, therefor, we prepare an conda env with python and provide scripts that are invoked in the package.json to launch that server.